### DISCUSSION TITLE
Unable to Connect- Connection Refused Error

### DISCUSSION THREAD

**Discussion by Ravua1992** [MichelNivard/gptstudio#162] [General]
Hello, 

I am unable to connect and get an error that the connection is refused.  can you kindly help ? 

![image](https://github.com/MichelNivard/gptstudio/assets/55991233/a5bdcc71-1502-41d1-8996-e9e1b24c07ce)

Thanks,

**Comment by calderonsamuel** [comment]
Please never share api keys in public!!

Check #179 to see the state of this issue. I'm closing this to keep the discussion in that link.
